back/app/services/location/service.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class Location::Service
+  def autocomplete(input, language = 'en')
```

Comment:
If `params` doesn't have the `:language` key, this `language` parameter will be `nil`. Is it what you expect?

I would maybe remove `= 'en'` to avoid any confusion as the default value is never used currently.
```suggestion
  def autocomplete(input, language)
```